### DISCUSSION TITLE
Wow! Thank you!

### DISCUSSION THREAD

**Discussion by PerfMonk** [mylinuxforwork/dotfiles#500] [Show and tell]
Just discovered and installed ML4W on my new fedora 41.  Wow !  It's real nice. I'm reading on how to customize some part of the hyprland and waybar config.  I would be very usefull to be able to "merge" and old config with the new one specially for keybinding.  I found myself a little bit lost without my usual keybinds.  I will do my own variation but it quite long to adapt if you dont want loose anything from the new ML4W.  
Thank you for the good work.  It very nice :=)

**Comment by mylinuxforwork** [comment]
@PerfMonk Thank you so much for your feedback.
